### DISCUSSION TITLE
学习工具清单

### DISCUSSION THREAD

**Discussion by leeduckgo** [NonceGeek/Move-Camp#24] [Materials]
- eth.build（无代码理解区块链原理）：https://sandbox.eth.build/
Web3 dApp Dev Camp Projects: https://github.com/WeLightProject/Web3-dApp-Camp/discussions/categories/projects
- CryptoZombie（僵尸游戏）:  
https://cryptozombies.io/zh/
- Openzepplin Contracts 合约库：
https://github.com/OpenZeppelin/openzeppelin-contracts/tree/master/contracts
- 智能合约案例合集：
https://solidity-by-example.org/
-Scaffold-Eth 脚手架：
https://github.com/scaffold-eth/scaffold-eth
- 区块链学习虚拟云主机：
http://os.doge.university/

**Comment by gofenix** [comment]
nice zombie
